### PR TITLE
SW-3633 Create Layer Selection Dropdown

### PR DESCRIPTION
**Pull request by charlie-tf**
image

### CODE REVIEW

File: src/components/common/MapLayerSelect/index.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  const classes = useStyles();
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const handleClick = (event?: React.MouseEvent<HTMLButtonElement, MouseEvent> | undefined) => {
+    setAnchorEl(event?.currentTarget ?? null);
+  };
+
+  const menuSections: { label: string; value: MapLayer }[][] = [
+    [
+      {
+        label: 'Planting Site',
```
Comment (karthikbtf) [inline src/components/common/MapLayerSelect/index.tsx:36]: use strings (for localization)

**Review by karthikbtf — APPROVED** [review]
Neat!
one request about strings

File: src/components/common/MapLayerSelect/index.tsx (hunk 1 above)
Comment (charlie-tf) [reply src/components/common/MapLayerSelect/index.tsx:36]: 🤦 Of course

File: .storybook/preview.tsx [hunk 2]
```diff
@@ -16,13 +16,27 @@ const preview: Preview = {
...
-      </StyledEngineProvider>
-    </ThemeProvider>
-  ),
+  (Story) => {
+    const [selectedLocale, setSelectedLocale] = useState('en');
+    const [activeLocale, setActiveLocale] = useState<string | null>(null);
+
+    return (
+      <ThemeProvider theme={theme}>
+        <LocalizationProvider
```
Comment (karthikbtf) [inline .storybook/preview.tsx:29]: Cool!

**Review by karthikbtf — APPROVED** [review]
LGTM!

File: .storybook/preview.tsx (hunk 2 above)
Comment (karthikbtf) [reply .storybook/preview.tsx:29]: do you need to `yarn build && yarn run storybook` as the vercel command?

Comment (charlie-tf) [reply .storybook/preview.tsx:29]: Maybe.. I'll give that a try, thanks

Comment (charlie-tf) [reply .storybook/preview.tsx:29]: adding `yarn generate-strings` did the trick :)

**Pull request merged**